flag/service/guest/update/update.go

```diff
@@ -0,0 +1,5 @@
+package update
+
+type Update struct {
+	Enabled string
```

Comment:
Does it make sense to make another nesting? Why not `UpdateEnabled`?

Reply:
Because this is how I structure data. I like structure and separate blocks. There is a fair chance that there is another property to be configured for updates and then we would have `UpdateEnabled` and `UpdateFoo` and I would not like this.

Reply:
Oh yeah and it is already like this in the kvm-operator.